tests/api-mongo.test.js

```diff
@@ -412,3 +416,26 @@ describe('MongoDB Words', function () {
     });
   });
 });
+
+describe('API Requests For Home Directory "/"', () => {
+  describe('/undefinedRoute', () => {
+    it('should return response status of 404', (done) => {
+      const route = '/undefinedRoute';
+      getApiUrlRoute(route).end((_, res) => {
+        expect(res.status).to.equal(404);
+      });
+      done();
+    });
+  });
+
+  describe('/', () => {
+    it('should return hello world!', (done) => {
+      getApiUrlRoute().end(async (_, res) => {
```

Comment:
Same thing here about moving `end` to a new line.

Also, I don't think you need to include the keyword `async`, you're not waiting for any promise to resolve inside the callback